iOS/BlueComputeApp/BlueComputeApp.xcodeproj/project.pbxproj

```diff
@@ -45,11 +45,9 @@
 		88BE50891D066AC30094A262 /* ApicOAuth2Module.swift in Sources */ = {isa = PBXBuildFile; fileRef = 88BE50881D066AC30094A262 /* ApicOAuth2Module.swift */; };
```

Comment:
All these are automatic changes that happened when I updated the libraries. Please ignore this whole file